src/containers/DoneCardContainer.tsx

```diff
@@ -0,0 +1,26 @@
+import { connect } from "react-redux";
+import {
+  getTodos,
+  postDone,
+  closePressedTodo,
+  editItemTodo,
+  saveEditItemTodo,
+} from "store/actions/index";
+import { TODO_STATUS } from "components/models";
+import { RootState } from "store/reducer/index";
+import Card from "components/Card";
+
+const stateToProps = (state: RootState) => {
+  return {
+    title: "Done",
+    todos: state.todos.data.filter((item) => item.state == TODO_STATUS.DONE),
```

Comment:
Read about memoization in react.
https://github.com/reduxjs/reselect

use createSelector from reselect for memoization.